app/jobs/document_proofing_job.rb

```diff
@@ -58,6 +60,8 @@ def perform(
           selfie_image: selfie_image || '',
           image_source: image_source(image_metadata),
           liveness_checking_enabled: liveness_checking_enabled,
+          user_uuid: applicant&.fetch(:uuid, SecureRandom.uuid),
+          uuid_prefix: applicant&.fetch(:uuid_prefix, nil),
```

Comment:
related to below comment about minimizing parameter mutation/defaulting in the job itself

```suggestion
          user_uuid: applicant&.fetch(:uuid),
```
```suggestion
          user_uuid: applicant&.fetch(:uuid),
          uuid_prefix: applicant&.fetch(:uuid_prefix),
```

Reply:
In my latest (b3712dc), I am still defaulting to nil, only to avoid a KeyError in the case of an old job that doesn't have the uuid pair.

Reply:
👍🏼 

can drop it in the future